lib/expeditor/command.rb

```diff
@@ -27,6 +28,16 @@ def start
       self
     end
 
+    def run
+      if not started?
+        @dependencies.each(&:start)
+        @executor = Concurrent::ImmediateExecutor.new
+        @normal_future.instance_variable_set(:@executor, @executor)
```

Comment:
Can't we avoid setting ivar from outside the instance by creating a method that sets `@executer` and call `safe_execute` or a `@executor` setter?